test/test_helpers/fake_fs.rb

```diff
@@ -5,9 +5,13 @@ def setup
       super
       ::FakeFS.clear!
       ::FakeFS.activate!
+      @fake_fs_called = true
     end
 
     def teardown
+      unless @fake_fs_called
```

Comment:
If this is called for each individual test, we should probably reset `@fake_fs_called = false` afterwards, otherwise we won't hit the error after the first test sets it up.

Reply:
I believe each test instantiates a new test testcase instance so that would prevent this from happening, but I'll add it anyway for clarity.